### ISSUE TITLE
Question: Storing cookies

### ISSUE THREAD

**Issue by arve0** [eliangcs/http-prompt#46]
Hi, nice app!
Can http-prompt store cookies automatically from a response like this?
```
Set-Cookie: author=f0a117c7-75a4-4abe-8617-1d3d06c4d2a7; path=/; httponly
Set-Cookie: author.sig=y4Kfr_MNE7RJmeL1x6vLhixHUAo; path=/; httponly
```
What I do now is copy/pasting (a bit painful):
```
Cookie:author=f0a117c7-75a4-4abe-8617-1d3d06c4d2a7;author.sig=y4Kfr_MNE7RJmeL1x6vLhixHUAo;
```

**Comment by eliangcs** [comment]
This is a very good feature request! I'll put it on the todo list.

**Comment by eliangcs** [comment]
Implemented in PR #71 and merged to master branch.
